### PR TITLE
Add overview component

### PR DESCRIPTION
**Pull request by sedroche**
Initial work on https://issues.jboss.org/browse/FH-4104 (Mobile overview screen updates)

Making view match wireframes outlined [here](https://docs.google.com/presentation/d/1B0RXEo8-J4TbHeCLC454N2y-LNN1-VobDdkHvp2UADM/edit#slide=id.g1244fbd6ce_2_117)

overview

- Refactored into a new component.
- updated buttons to default and primary
- Consistent layout

### CODE REVIEW

File: ui/public/scripts/controllers/mobileoverview.js [hunk 1]
```diff
@@ -39,23 +43,71 @@ angular.module('mobileControlPanelApp').controller('MobileOverviewController', [
...
+              canView: function() {
+                return AuthorizationService.canI(
+                  'services',
+                  'create',
+                  projectContext.projectName
+                );
+              }
+            },
+            {
+              label: 'Provision Catalogue Service',
```
Comment (david-martin) [inline ui/public/scripts/controllers/mobileoverview.js]: better to go with the American spelling that's used elsewhere in OpenShift -> "Catalog"

**Comment by david-martin** [comment]
@sedroche Do you want to include Card updates in this PR too? The text looks centered in the mockup for each card, compared to the left aligned for service id & type.

File: ui/public/scripts/controllers/mobileoverview.js (hunk 1 above)
Comment (sedroche) [reply ui/public/scripts/controllers/mobileoverview.js]: Yep

**Comment by sedroche** [comment]
@david-martin I'll align the text in this PR

**Comment by sedroche** [comment]
@david-martin Have updated to match your feedback.

centered-text

catalog

**Pull request merged**